### ISSUE TITLE
Beaming of web app is failing on Chromecast 

### ISSUE THREAD

**Issue by dkonovalov** [ConnectSDK/Connect-SDK-Cordova-Plugin#59]
I try to use ConnectSDK following your example code (basic cordova app with ConnectSDK plugin). I have Chromecast V2, which is standard. I am able to enumerate cast devices on the network and connect to my chromecast, but when I try to beam web app, I end up in the error callback, with a error message like this: 
"Exception calling launchWebApp: java.lang.NullPointerException: Attempt to invoke interface method 'void com.connectsdk.service.capability.WebAppLauncher.launchWebApp(java.lang.String, com.connectsdk.service.sessions.WebAppSession$LaunchListener)' on a null object reference". 
Chromecast device I am using is standard, and I am able to beam WebApp on it using google own casting example. I assume that problem is that the device reported as missing webapplaunch capability, but I might be wrong. Could you please provide a little information about possible troubleshooting steps here?

**Comment by federicotarantino** [comment]
What webapp-id are you using in casting example?

**Comment by dkonovalov** [comment]
Hi. I used "C975EE6E" as a webapp-id.

**Comment by federicotarantino** [comment]
Are you verified if this webapp exists?

**Comment by dkonovalov** [comment]
Of course. As stated in original post, I am able to connect to this app by using Google demo javascript casting client. And, I am getting the same error result with ConnectSDK if I use any of the Chromecast demo applications I was able to find, so I do not think it is app related, but rather should be an SDK problem.
